fastrpc.cmake

```diff
@@ -74,11 +74,9 @@ else()
 endif()
 
 set(ADSPMSGD ${HEXAGON_SDK_ROOT}/${SDKLIB}/common/adspmsgd/ship/UbuntuARM_${RELEASE}/adspmsgd.a)
-set(RPCMEM ${HEXAGON_SDK_ROOT}/${SDKLIB}/common/rpcmem/UbuntuARM_${RELEASE}/rpcmem.a)
```

Comment:
This is no longer needed as the source file that goes into the library is compiled into the LINUX_LIB and LINUX_APP macros.  The Issue with library reference is that the APQ8096 platform compiler is using softfp and the pre-compiled library uses hardfp.  This causes a link time error for the apps processor lib/application.